sample/src/main/java/io/ona/kujaku/sample/activities/LowLevelLocationAddPointMapView.java

```diff
@@ -30,34 +24,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
         kujakuMapView = findViewById(R.id.kmv_lowLevelLocationAddPointMapView_mapView);
 
-        Button doneBtn = findViewById(R.id.btn_lowLevelLocationAddPointMapView_doneBtn);
-        Button goToMyLocationBtn = findViewById(R.id.btn_lowLevelLocationAddPointMapView_myLocationBtn);
+        kujakuMapView.showCurrentLocationBtn(true);
 
-        doneBtn.setOnClickListener(new View.OnClickListener() {
+        kujakuMapView.addPoint(false, new AddPointCallback() {
```

Comment:
Why remove this low-level API illustration and switch it with a high-level API illustration in a low-level API intended illustration?

Reply:
Reverted to previous implementation.